Documentation/migration-scripts/shell-scripts/update-org-code.sh

```
@@ -0,0 +1,187 @@
+log "🧾 Observation Status Tables: ${#observation_status_ids[@]}"
+log "🧾 Observation Questions Tables: ${#observation_question_ids[@]}"
+
+# === Normalize function ===
+normalize_table() {
+  local table=$1
+  local column=$2
+
+  log "🔎 Checking $table.$column for normalization needs..."
+
+  # Count rows needing normalization
+  local count=$(psql -h "$PGHOST" -p "$PGPORT" -d "$PGDBNAME" -U "$PGUSER" -Atc \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)